calculator.rb

```diff
@@ -0,0 +1,150 @@
+# calculator.rb
+# https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2FAda-C10%2Fcalculator&sa=D&usd=2&usg=AFQjCNF8YsY1ij6pguWOvbMUgTJO86XQ7Q
+
+$known_adds = ["+", "add", "plus"]
```

Comment:
We'll get into this as a class later, but **never ever ever** use global variables.  It's a bad practice and can make code that's very hard to debug.

Reply:
Haha... well learning about scope and constants today was definitely a game changer for me!